geomstats/geometry/lie_group.py

```diff
@@ -255,22 +253,14 @@ def exp(self, tangent_vec, base_point=None, point_type=None):
         result : array-like, shape=[n_samples, {dimension,[n,n]}]
             The exponentiated tangent vector
         """
-        if point_type is None:
-            point_type = self.default_point_type
-
         identity = self.get_identity(point_type=point_type)
-        identity = self.regularize(identity, point_type=point_type)
+        identity = gs.to_ndarray(
+            identity,
+            to_ndim=geomstats.vectorization.POINT_TYPES_TO_NDIMS[point_type])
```

Comment:
Maybe that's what needs be done above as well (see previous comments)?

Reply:
Yes, good eye!